index.php

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE HTML>
+<html lang='en'>
+    <head>
+        <meta charset='UTF-8'>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="theme-color" content="#111111">
+        <title>The Real MVP</title>
+        <link rel="icon" type="image/x-icon" href="./app/images/favicon.ico">
+        <link type='text/css' rel='stylesheet' href='./app/css/normalize.css' />
+        <link type='text/css' rel='stylesheet' href='./app/css/style.css' />
+    </head>
+    <body>
+        <?php
+
+use TheRealMVP\DBImport;
+use TheRealMVP\DisplayData;
+use TheRealMVP\GetAPI;
+use TheRealMVP\TeamHydrator;
+
+require_once './vendor/autoload.php'; ?>
+        <header>
+            <h1>The Real MVP</h1>
+        </header>
+        <main>
+                <?php 
+                $api = new GetAPI();
```

Comment:
We don't want to trigger the API import on every page load, would be better if you added a separate file (just called import.php or something like that) that does nothing else apart from import the data from the API.

Reply:
That way whever you need to refresh the data, you just go to /import.php and it does it for you

Reply:
As the GetAPI object now only creates a connection - the DBImport actually gets the api data. So do we still need to do this?